src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/induction/FutureWorkInterestsEntity.kt

```diff
@@ -42,8 +46,7 @@ class FutureWorkInterestsEntity(
   @field:NotNull
   var reference: UUID? = null,
 
-  @OneToMany(cascade = [CascadeType.ALL], orphanRemoval = true, fetch = FetchType.LAZY)
-  @JoinColumn(name = "work_interests_id", nullable = false)
+  @OneToMany(mappedBy = "parent", cascade = [CascadeType.ALL], orphanRemoval = true, fetch = FetchType.LAZY)
   var interests: MutableList<WorkInterestEntity>? = null,
```

Comment:
We now have a one-to-many "parent/child" bidirectional relationship. When a "child" is modified (including inserted/removed), we are able to use the JPA lifecycle callback methods (e.g. `@PreUpdate`) to update `updatedAt` on the "parent" entity.